### PR TITLE
fix: azure,bamboo,github,jira distinct primary key

### PR DESCRIPTION
**Pull request by abeizn**
### Summary
fix: azure,bamboo,github,jira distinct primary key

### Does this close any open issues?
Related to #6768

### Screenshots
image

### Other Information
Any other information that is important to this PR.

### CODE REVIEW

File: grafana/dashboards/GitHub.json [hunk 1]
```diff
@@ -3464,7 +3516,10 @@
             "$__all"
           ]
         },
-        "datasource": "mysql",
+        "datasource": {
```
Comment (Startrekzky) [inline grafana/dashboards/GitHub.json]: This should be removed

File: grafana/dashboards/Jira.json [hunk 2]
```diff
@@ -138,15 +165,20 @@
         "text": {},
         "textMode": "auto"
       },
-      "pluginVersion": "8.0.6",
+      "pluginVersion": "9.5.15",
       "targets": [
         {
+          "datasource": {
+            "type": "mysql",
+            "uid": "P430005175C4C7810"
```
Comment (Startrekzky) [inline grafana/dashboards/Jira.json]: This should be reverted

File: grafana/dashboards/Jira.json [hunk 3]
```diff
@@ -222,15 +271,20 @@
...
```
Comment (Startrekzky) [inline grafana/dashboards/Jira.json]: THis should be reverted

File: grafana/dashboards/Jira.json [hunk 4]
```diff
@@ -330,15 +402,21 @@
...
+          "mode": "multi",
+          "sort": "none"
         }
       },
       "pluginVersion": "8.0.6",
       "targets": [
         {
+          "datasource": {
+            "type": "mysql",
+            "uid": "P430005175C4C7810"
```
Comment (Startrekzky) [inline grafana/dashboards/Jira.json]: This should be reverted.

File: grafana/dashboards/Jira.json [hunk 5]
```diff
@@ -426,9 +502,13 @@
         "text": {},
         "textMode": "auto"
       },
-      "pluginVersion": "8.0.6",
+      "pluginVersion": "9.5.15",
       "targets": [
         {
+          "datasource": {
+            "type": "mysql",
+            "uid": "P430005175C4C7810"
```
Comment (Startrekzky) [inline grafana/dashboards/Jira.json]: Same

File: grafana/dashboards/Jira.json [hunk 6]
```diff
@@ -532,15 +613,21 @@
...
+          "mode": "single",
+          "sort": "none"
         }
       },
       "pluginVersion": "8.0.6",
       "targets": [
         {
+          "datasource": {
+            "type": "mysql",
+            "uid": "P430005175C4C7810"
```
Comment (Startrekzky) [inline grafana/dashboards/Jira.json]: Same

File: grafana/dashboards/Jira.json [hunk 7]
```diff
@@ -639,9 +736,14 @@
         "text": {},
         "textMode": "auto"
       },
-      "pluginVersion": "8.0.6",
+      "pluginVersion": "9.5.15",
       "targets": [
         {
+          "datasource": {
+            "type": "mysql",
+            "uid": "P430005175C4C7810"
```
Comment (Startrekzky) [inline grafana/dashboards/Jira.json]: Same.

File: grafana/dashboards/Jira.json [hunk 8]
```diff
@@ -726,9 +843,13 @@
...
```
Comment (Startrekzky) [inline grafana/dashboards/Jira.json]: Same.

File: grafana/dashboards/Jira.json [hunk 9]
```diff
@@ -817,25 +944,36 @@
...
+        },
+        "xTickLabelRotation": 0,
+        "xTickLabelSpacing": 0
       },
       "pluginVersion": "8.0.6",
       "targets": [
         {
+          "datasource": {
+            "type": "mysql",
+            "uid": "P430005175C4C7810"
```
Comment (Startrekzky) [inline grafana/dashboards/Jira.json]: Same.

File: grafana/dashboards/Jira.json [hunk 10]
```diff
@@ -921,6 +1057,10 @@
       "steppedLine": false,
       "targets": [
         {
+          "datasource": {
+            "type": "mysql",
+            "uid": "P430005175C4C7810"
```
Comment (Startrekzky) [inline grafana/dashboards/Jira.json]: Same.

**Review by Startrekzky — CHANGES_REQUESTED** [review]

File: grafana/dashboards/Jira.json (hunk 10 above)
Comment (abeizn) [reply grafana/dashboards/Jira.json]: all fixed

**Review by Startrekzky — APPROVED** [review]
LGTM

**Pull request merged**